tools/generate-sponsors.ts

```diff
@@ -24,6 +25,10 @@ const graphqlQuery = `{
   }
 }`;
 
+const excludedIds = new Set([
+  '53kzxy4v-07wlr6mr-o9epmj9n-o8agdbe5', // Josh Goldberg
```

Comment:
I don't mind being here, but the list of contributors overflows unless we remove one... 😄 

Also, my GraphQL competency is pretty weak and I couldn't figure out the right `: not`; otherwise I'd embed this in that query...

Reply:
> but the list of contributors overflows unless we remove one

Wait - won't this be a problem in future if we have more sponsors?

Reply:
Yeah. It's not the end of the world, just a single square on a next line. I can always add myself back 😄